README.md

```diff
@@ -133,6 +133,12 @@ With that setup, you could do the following in the terminal:
 - **Initialize a new project:** `npx gulp init`
 - **Update USWDS static assets and recompile:** `npx gulp update`
 
+### Usage tips
+- **Use `init` only once.** The `init` task is meant for initializing the design system on a project. Since it will overwrite project files (like settings files and the Sass entry point), use it sparingly and don't use it for updating the design system on a project, or at any point after you've customized your settings fies.  
```

Comment:
```suggestion
- **Use `init` only once.** The `init` task is meant for initializing the design system on a project. Since it will overwrite project files (like settings files and the Sass entry point), use it sparingly and don't use it for updating the design system on a project, or at any point after you've customized your settings files.  
```

Whoops, found a small typo after I approved! This one _should_ be fixed before merging :)